index.html

```diff
@@ -3,11 +3,54 @@
 <head>
   <meta charset="UTF-8">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <!-- Don't forget to change the title and connect the CSS file -->
-  <title>Project Library</title>
+  <link rel="stylesheet" href="style.css">
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+  <title>Digital Library</title>
 </head>
 <body>
-  <h1>Project Library</h1>
-  <!-- Don't forget to connect the JavaScript file -->
+
+  <header>
+    <div class="hero.text">
+      <h1>Welcome to Our Library</h1>
+      <h5>Explore - Learn -  Grow</h5>
+    </div>
+    <div class="sortButtons">
+      <!-- <p>alphabetical sorting</p> -->
+        <div id="az-za">
+          <button id="az">A-Z</button>
+          <button id="za">Z-A</button>
+        </div>
+        <!-- <p>Sort Genre</p> -->
+          <select id="genreDropdown">
+            <option value="all">All genres</option>
+            <option value="Fantasy">Fantasy</option>
+            <option value="Fiction">Fiction</option>
+            <option value="Mystery">Mystery</option>
+            <option value="Horror">Horror</option>
+            <option value="Science Fiction">Science fiction</option>
+            <option value="Adventure">Adventure</option>
+            <option value="Dystopian">Dystopian</option>
+          </select>        
+      <!-- <p>Most readed</p> -->
+        <div>
+          <button id="btnRate">Rate</button>
+        </div>
+    </div> 
+  </header>
+
+  <main class="main">
+    <div id="container">
+      <!-- here is where our books go -->
+    </div>
+  </main>
+  
+  </footer>
+    <div class="footer-content">
+      <p> 2024 Arnau Vidal, Lorenzo Dastoli</p>
+      <img src="books-images/logo.svg" alt="Technigo" class="logo">
+    </div>
+  </footer>
```

Comment:
Be careful, you're using the footer tag with just closing tags. Also I would remove one of these elements, keep either the ` <footer></footer>` or the ` <div class="footer-content">` here 👍

Reply:
I saw it while trying to adjust the footer, I just forgot to uppdate it. Good eye!